azure-mgmt-storage/src/main/java/com/azure/management/storage/Action.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.management.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
```

Comment:
why this new file not in preview generation? swagger change?

Reply:
This is the fix on non-required constant. This enum got only one value. But since it is non-required, user could still choose between `null` and `Action.ALLOW`.